### ISSUE TITLE
Yandex Maps JS API: Invalid API key

### ISSUE THREAD

**Issue by YauhenDev** [R1ZEN/react-yandex-maps#64]
Не нашел куда необходимо добавлять ключ который получил в [кабинета разработчика](https://developer.tech.yandex.ru)?

**Comment by snowinmars** [comment]
@YauhenDev 
В [предыдущей репе](https://github.com/gribnoysup/react-yandex-maps/issues/59) сказано, что

```
<YMaps
  enterprise
  query={{
    apikey: 'ahjgasgdjhasgd',
  }}>
...
</YMaps>
```

**Comment by BulochkaBU** [comment]
А с такой ошибкой сталкивались?
(Yandex Maps JS API): Invalid API key - ymapsInit | @ | ?onload=__yandex-map…bee8-cc2133ef42a3:1
подключала также
`<YMaps
  query={{
    apikey: 'ahjgasgdjhasgd',
  }}>
...
</YMaps>`

**Comment by snowinmars** [comment]
@BulochkaBU , ну, типа, ключ неверный, надо получить верный.

Ваще, карты будут работать и без ключа. Это дополнительная штука для прода, не более.

**Comment by BulochkaBU** [comment]
Да, без ключа работает. Я еще раз сгнерировала ключ и ошибки больше не возникало.
